config.json

```diff
@@ -322,6 +322,10 @@
       "mastercard",
       "visa"
     ],
+    "supported_instrument_types": [
```

Comment:
Payment method in this context indicates the payment provider for the vaulted instrument rather than the instrument type. I think `supported_payment_methods` is best name for this variable.

Reply:
done